scripts/dockerfiles/alplatform

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+pfx=${1:-TARGET}
+
+eval platform=\${${pfx}PLATFORM}
+
+die() {
+    echo $@ >&2
+    exit 1
+}
+
+case $platform in
+    linux/amd64) echo x86_64 ;;
+    linux/386|linux/386/*) echo i686 ;;
```

Comment:
amazonlinux (x86_64) apparently has i686 packages in the package archive too.  I haven't tested that on a 32 bit machine (there's no linux/386 entry in the amazonlinux:2 manifestlist) - but I just figured I'd add it while I was here.  If there _was_ to be a 32 bit intel AL2, it would use "i686" afaics.